src/nile/core/call_or_invoke.py

```diff
@@ -5,13 +5,18 @@
 
 from nile import deployments
 from nile.common import GATEWAYS, prepare_params
+from nile.core import account
 
 
 def call_or_invoke(
     contract, type, method, params, network, signature=None, max_fee=None
 ):
     """Call or invoke functions of StarkNet smart contracts."""
-    address, abi = next(deployments.load(contract, network))
+    if isinstance(contract, account.Account):
+        address = hex(contract.address)
+        abi = contract.abi_path
+    else:
+        address, abi = next(deployments.load(contract, network))
```

Comment:
```suggestion
    else:
        address, abi = next(deployments.load(contract, network))
        address = hex(address)
```
I believe the `address` should also be converted to `hex`. When I try to script a basic `get_balance` query, I'm receiving a type error:

`TypeError: expected str, bytes or os.PathLike object, not int`

Reply:
I added it outside the if/else for both branches before calling the starknet cli.

Reply:
Perfect!